packages/algjulia-service/src/AlgebraicJuliaService.jl

```diff
@@ -7,27 +7,72 @@ using Reexport
 # the intent is that this is an interface for AlgebraicJulia code to interoperate with CatColab 
 @data AlgebraicJuliaIntegration begin
     ThDecapode()
-end
-export ThDecapode
+end; export ThDecapode
```

Comment:
TBH, I prefer the previous style, which feels like more idiomatic Julia despite using slightly more whitespace.